src/helpers/utils.js

```diff
@@ -15,3 +15,7 @@ export const mergeUnion = (sourceObject, overrideObject) => {
 
   return { ...sourceObject, ...Object.fromEntries(new Map(union)) }
 }
+
+export const addEllipsisToLongString = (str, maxChars) => {
```

Comment:
Do you know about "default values" in function parameters? I think in this case it could be useful!

Reply:
Hey John, left it without a default as if we extend it for use in other columns doesn't seem like it is appropriate, pls leave your thoughts if this shouldn't be the case!